### PR TITLE
Home screen fix

### PR DESCRIPTION
**Pull request by dkyeremeh**
Addresses #878 

- Fixed the top and bottom cutoff
- Changed the background to accommodate the virtual scroll

@tananaev I've attached a video of how it looks now

https://user-images.githubusercontent.com/52928432/126792449-45722c5d-b303-4a01-a2d7-01b383dafb97.mov

### CODE REVIEW

File: modern/src/common/selectors.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
 export const getIsAdmin = (state) => state.session.user?.administrator;
 
 export const getUserId = (state) => state.session.user?.id;
+
+export const getDevices = (state) => Object.values(state.devices.items);
+
+export const getItemPosition = (itemId) => (state) => state.positions.items[itemId];
```
Comment (tananaev) [inline modern/src/common/selectors.js:7]: Can we rename with to just `getPosition`? And we don't need to use `itemId` here. It's just position id.

File: modern/src/DevicesList.js [hunk 2]
```diff
@@ -145,7 +151,9 @@ const DeviceView = ({ updateTimestamp, onMenuClick }) => {
             height={height}
             itemCount={items.length}
             itemData={{ items, onMenuClick }}
-            itemSize={72 + 1}
+            itemSize={72}
+            overscanCount={10}
+            innerRef={listInnerEl}
```
Comment (tananaev) [inline modern/src/DevicesList.js:156]: Why do we need this parameter?

Comment (dkyeremeh) [reply modern/src/DevicesList.js:156]: @tananaev It's to allow styling of the list container. We need space at the bottom and top of the list. Putting the space anywhere else result in the list item being cut. This is the problem you pointed out in point 1 of https://github.com/traccar/traccar-web/issues/878

File: modern/src/common/selectors.js (hunk 1 above)
Comment (dkyeremeh) [reply modern/src/common/selectors.js:7]: Noted

**Comment by dkyeremeh** [comment]
@tananaev PR updated

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**